Core/Product Request/PaymentTransactionObserver.swift

```diff
@@ -79,19 +79,23 @@ public class PaymentTransactionObserver: NSObject, SKPaymentTransactionObserver
     }
     
     public func paymentQueueRestoreCompletedTransactionsFinished(_ queue: SKPaymentQueue) {
-        // Check to see if the returned queue as any transactions.  If there are no
-        // transactions, it means the user has not made any purchases yet, so there is
-        // nothing to restore.
-        guard queue.transactions.count > 0 else {
-            // There were no transactions for this user
-            let localizedErrorMessage = NSLocalizedString("Sorry, no purchased products were found. If you have questions, please contact support@blackpixel.com.", tableName: nil, bundle: Bundle(for: PaymentTransactionObserver.self), value: "", comment: "no previous transactions were found on the users current iTunes account error message")
-            
-            delegate?.transactionDidFail(localizedErrorMessage: localizedErrorMessage)
+        // fetch support email or nil if it doesn't exist
+        if let info = Bundle.main.infoDictionary,
+           let supportEmail = info["HGSupportEmail"] as? String {
```

Comment:
What do you think about using `IAPSupportEmail` as the key instead?

Reply:
the key is common to the application, not specifically targeted towards IAP since we also use it in MAS sales as well.  I'm ok with changing the key name.  It just needs to be identifiable so we have the option to configure during the build phase.

Reply:
Yea, maybe `IAPKitSupportEmail` would be better.